libs/components/src/tabs/tab-item.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+  selector: "bit-tab-item",
+  templateUrl: "./tab-item.component.html",
+})
+export class TabItemComponent {
+  @Input()
+  route: string; // ['/route']
```

Comment:
My personal preference are to keep these on the same line. If they only have a single decorator

```suggestion
  @Input() route: string; // ['/route']
```

Reply:
Alrighty.